empire/core/scenario_random.py

```diff
@@ -1005,7 +1005,10 @@ def check_scenarios_exist_and_copy(run_config: EmpireRunConfiguration):
                 run_config.tab_file_path,
             )
         else:
-            shutil.copyfile(run_config.scenario_data_path / file, run_config.tab_file_path / file)
+            try:
+                shutil.copyfile(run_config.scenario_data_path / file, run_config.tab_file_path / file)
+            except shutil.SameFileError:
+                pass

```

Comment:
Workaround for å unngå kopiere over filene hver gang man kjører out-of-sample

Reply:
Kontekst så kaster funksjonen SameFileError om man prøver å klippe-lime til samme lokasjon